app/upgrades/v1_6.go

```diff
@@ -16,7 +17,12 @@ func V010600UpgradeHandler(
 	appKeepers *keepers.AppKeepers,
 ) upgradetypes.UpgradeHandler {
 	return func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
-		// no action yet.
+		// Update dust threshold configuration for all zones
+		appKeepers.InterchainstakingKeeper.IterateZones(ctx, func(index int64, zone *icstypes.Zone) (stop bool) {
+			zone.DustThreshold = 1_000_000
```

Comment:
We intentionally _don't_ want to set this to 1_000_000 across the board.

Atom can be much lower. At 12 bucks it can reasonably be 250_000.

For others, it wants to be higher:

Osmo: 1_000_000
Juno: 2_000_000
Regen, Stars, Somm: 5_000_000

Reply:
hmmmmm see this would be an interesting case to pull data from Osmosis.  By using its prices, we could have a defined dust threshold, we could call it $1.

Adding this to a thread in osmosis development telegram...

Reply:
Should I make a proposal that given a list of chain-id and the dust threshold value, then set it into each zone.

Reply:
tbh I think it would be way clearer to pull prices from osmo, then we don't need governance, and dust could be set to a clear value.

Reply:
the only other solutions I can see are:

* lots of gov props for price movement (bad)
* update these values with every upgrade (bad)

so, I think it is better to pull prices from osmosis.

Reply:
there is some (idk exactly how) way to query osmosis for time-weighted average price. 

We could pull prices form osmosis epochly, and then dust would always have a reasonable value.

Reply:
It doesn't need to be strictly price related - it's not critical it's exact. It's more about relative value. With claims we actually already have enough data to determine the relative value between zones.

Reply:
Personally I would say that the definition of dust is always price related, wdyt?

Reply:
It is, but it's not critical it is exact. A complex piece of logic fetching a twap is overkill. We already have the spot price for each zone with respect to osmo for each epoch, and that is sufficient.